tests/test_masked_inference_wsi_dataset.py

```diff
@@ -27,11 +27,10 @@
 _, has_cim = optional_import("cucim", name="CuImage")
 _, has_osl = optional_import("openslide")
 
-FILE_KEY = "wsi_img"
+FILE_KEY = "wsi_generic_tiff"
```

Comment:
will this close https://github.com/Project-MONAI/MONAI/pull/5878, please clarify

Reply:
Yes, it does!